static/panes/compiler.js

```diff
@@ -915,6 +916,28 @@ Compiler.prototype.onGccDumpFiltersChanged = function (id, filters, reqCompile)
         this.treeDumpEnabled = (filters.treeDump !== false);
         this.rtlDumpEnabled = (filters.rtlDump !== false);
 
+        this.dumpFlags = '';
```

Comment:
not sure this makes sense to me - am I missing something?

If you check multiple options its going to produce a string like:
`-address-slim-raw`

Reply:
It is correct usage.  From the docs

> If the ‘-options’ form is used, options is a list of ‘-’ separated options which control the details of the dump. Not all options are applicable to all dumps; those that are not meaningful are ignored.

Reply:
ah, ok - I was looking at the docs for `-fdump-rtl`, which doesn't have the same note about combining in that way.

Reply:
> ah, ok - I was looking at the docs for `-fdump-rtl`, which doesn't have the same note about combining in that way.

I can check it to be sure, but as I understand it, they all use the same machinery for option parsing.

Reply:
Confirmed.  It's a bit of a free-for-all option.  The only part that is fixed is `-fdump-`, everything else that follows is handled by magic.